ecl/eclccserver/eclccserver.cpp

```diff
@@ -963,7 +1040,13 @@ class EclccServer : public CInterface, implements IThreadFactory, implements IAb
     virtual bool onAbort() 
     {
         running = false;
-        if (queue)
+        Linked<IJobQueue> currentQueue = queue;
```

Comment:
This default value isn't safe.  E.g. it could be released (and set to null) before the critical section is entered.

Reply:
yes, that was a mistake, it is set in CS, shouldn't have a default